client/components/Events/EventHistory.jsx

```diff
@@ -167,6 +171,17 @@ export class EventHistory extends React.Component {
                                                     </a>
                                                 </div>
                                             }
+
+                                            {historyItem.operation === HISTORY_OPERATIONS.CREATED_FROM_PLANNING &&
+                                            get(historyItem, 'update.created_from_planning') &&
+                                                <div className="history-list__link">
+                                                    <a onClick={this.closeAndOpenDuplicate.bind(this,
+                                                        historyItem.update.created_from_planning, 'planning')}>
```

Comment:
Use the `ITEM_TYPE` constant wherever you are passing through an item type string